### PR TITLE
GHA Notifier

### PR DESCRIPTION
**Pull request by sarinali**
# Description

Monitoring version made for Windows and Mac

## Type of change

- [x] Other Changes: Monitoring Feature

# How does this resolve / make progress on that bug?

After each push, PR there will be artifacts uploaded and available for you to view. A slack message will send to notify you that the job is done as well.

# Screenshots / Explanations

image

# Comments / Concerns

https://docs.google.com/document/d/1bpxx9eLmtZ68dc_xE-aXPOepqzsrPPtPLX3GpuN3UMc/edit (some documentation stuff and basic step by step process implementation)

### CODE REVIEW

File: .github/workflows/smoke.yml [hunk 1]
```diff
@@ -96,11 +96,52 @@ jobs:
         run: |
           mv ./ci_pyproject_headed.toml ./pyproject.toml;
           pipenv run pytest --fx-executable=/Volumes/Firefox/Firefox.app/Contents/MacOS/firefox -n 4 . || TEST_EXIT_CODE=$?
-          mv -r artifacts/* artifacts-mac/ || true
+          mv artifacts/* artifacts-mac/ || true
```
Comment (ben-c-at-moz) [inline .github/workflows/smoke.yml:99]: You're right, `-r` is fake. Can we change it to `-n`? I don't want to clobber anything.

File: tests/pdf_viewer/test_pdf_download.py [hunk 2]
```diff
@@ -75,4 +75,3 @@ def test_pdf_download(
     print(
         f"Test passed: The file {file_name} has been downloaded and is present at {saved_pdf_location}."
     )
-    driver.quit()
```
Comment (ben-c-at-moz) [inline tests/pdf_viewer/test_pdf_download.py:78]: Why remove this?

**Review by ben-c-at-moz — CHANGES_REQUESTED** [review]
One change doesn't make sense to me, but other than that, this looks great. (I suspect the change is coming from a linter or something)

File: tests/pdf_viewer/test_pdf_download.py (hunk 2 above)
Comment (sarinali) [reply tests/pdf_viewer/test_pdf_download.py:78]: I think linting deleted this which is super weird, reverting now

File: .github/workflows/smoke.yml (hunk 1 above)
Comment (sarinali) [reply .github/workflows/smoke.yml:99]: ok! on it

**Review by ben-c-at-moz — APPROVED** [review]

**Pull request merged**